Documentation/botr/readytorun-overview.md

```diff
@@ -0,0 +1,335 @@
+Managed Executables with Native Code
+===
+
+# Motivation
+
+Since shipping the .NET Runtime over 10 years ago, there has only been one file format which can be used to distribute and deploy managed code components: the CLI file format. This format expresses all execution as machine independent intermediate language (IL) which must either be interpreted or compiled to native code sometime before the code is run. This lack of an efficient, directly executable file format is a very significant difference between unmanaged and managed code, and has become more and more problematic over time. Problems include
```

Comment:
Please add a colon after 'include' (and likewise for each of the other lists below).